tasks/eva_2130/remove_duplicates_in_study.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+import pymongo
+from urllib.parse import quote_plus
+
+
+def find_duplicates_and_remove_them(mongo_user, mongo_password, mongo_host, mongo_database, assembly_accession,
+                                    contig_list, study_list, dry_run):
+    duplicates_to_remove_commands = []
+    with get_mongo_connection_handle_url(
+            username=mongo_user,
+            password=mongo_password,
+            host=mongo_host
+    ) as accessioning_mongo_handle:
+        sve_collection = accessioning_mongo_handle[mongo_database]["submittedVariantEntity"]
+        cursor = sve_collection.find({
+            "seq": assembly_accession, "contig": {"$in": contig_list}, "study": {"$in": study_list}
+        })
+        for record in cursor:
+            variants_with_accessions = list(accessioning_mongo_handle[mongo_database]["submittedVariantEntity"].find({'accession': record['accession']}))
+            if len(variants_with_accessions) != 1:
+                print('Found %s duplicates for accession %s' % (len(variants_with_accessions), record['accession']) )
+            for variant in variants_with_accessions:
+                if variant['contig'] != record['contig']:
```

Comment:
I expected some more checks to identify duplicates. This will delete multimap accessions too. I guess it's fine if we know this script shouldn't be run on dbsnp collections. This script will become obsolete if we ever store multimap variants from our remapping.

I suggest considering checking that `variant['contig']` and `record['contig']` are synonyms according to the assembly report, and also checking that the identifying fields are the same (asm, start, type). If this script is really meant to be run only for EVA-2130, then my suggestion might be overkill, though.

Reply:
The code in EVA task is only meant for this task and this task alone. Even if I wanted to reuse the same code I would copy/paste it. I can add the test on identifying fields but adding extra complexity of loading the assembly reports seems unnecessary.